.circleci/config.yml

```diff
@@ -88,8 +88,8 @@ jobs:
             docker create -v /src/botoenv/tests/ --name service_tests alpine:3.4 /bin/true
             docker cp $(pwd)/botoenv/tests service_tests:/src/botoenv/
             eval $(aws ecr get-login --no-include-email)
-            docker pull ${AWS_ECR_DOMAIN}/botoenv:${CIRCLE_SHA1}
-            docker run -it  --volumes-from service_tests ${AWS_ECR_DOMAIN}/botoenv:${CIRCLE_SHA1} test
+            docker pull ${AWS_ECR_DOMAIN}/python-library:${CIRCLE_SHA1}
+            docker run -it  --volumes-from service_tests ${AWS_ECR_DOMAIN}/python-library:${CIRCLE_SHA1} test
```

Comment:
Ultra-nit:
Why two spaces between `-it` and `--volumes-from`? Looks like a typo.

Reply:
Probably but an auto-generated typo :)